src/sentry/integrations/migrate.py

```diff
@@ -21,7 +13,7 @@ def __init__(self, integration, organization):
     def call(self):
         for project in self.projects:
             for plugin in plugins.for_project(project):
-                if plugin.slug not in DISABLEABLE_PLUGINS:
+                if plugin.slug != self.integration.provider:
```

Comment:
so this is confusing af, but i think they do match for integration / plugin slug:
https://github.com/getsentry/sentry-plugins/blob/master/src/sentry_plugins/vsts/plugin.py#L17
https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/vsts/integration.py#L252

what doesn't match, for some reason, is the REPO provider: https://github.com/getsentry/sentry-plugins/blob/master/src/sentry_plugins/vsts/plugin.py#L165 (but matte already accounts for that below)

right?? or am i missing something else

Reply:
but vsts did support repos before

Reply:
Could we make the not-matching bits go away for future us?

Reply:
@macqueen Yea, you're correct.

`VstsPlugin#slug == VstsIntegrationProvider#key`. (`vsts`)

`Repository#provider` **will not** match either of the above. (`visualstudio`)

Reply:
@markstory yeah, but that would involve a database migration + other changes and it would be for an integration that we're planning on deprecating anyway. so definitely outside of the scope of this pr and likely not worth the effort given the deprecation